assets/src/edit-story/components/panels/link.js

```diff
@@ -47,10 +48,13 @@ function LinkPanel({ selectedElements, pushUpdateForObject }) {
   const inferredLinkType = useMemo(() => inferLinkType(selectedElement), [
     selectedElement,
   ]);
-  const DEFAULT_LINK = createLink({ url: null, icon: null, desc: null });
+  const defaultLink = useMemo(
```

Comment:
Why did we move this inside to start with?

Reply:
Because of this: #723. I still don't understand why that was needed in the first place, but at least I'm trying to make it into more of a constant, instead of thrashing all of the callbacks.

Reply:
Good idea with useMemo, I should have done that.

I didn‘t spend too much time investigating the underlying issue, but anything outside the imported createLink function was undefined - unless it was called within this component rather than outside.

Reply:
Let's merge this PR and look into the import problem separately. For now we need to fix the link declaration here.